Production website hard restarting from export to csv

Seeing the following errors on the production website. Through a bit of debugging I believe that this is caused by the *Export to CSV* button. After clicking that the website becomes unresponsive. The deployed version of the website is 9c56bd975b2081dddc6c603f9d5d318b247e562c

Delete the branch `hotfix/jmadden173/remove-csv-export` when completed.

```
March 4, 2026, 11:45
	
[2026-03-04 19:45:52 +0000] [1] [INFO] Handling signal: term
	
dirtviz-api
March 4, 2026, 11:42
	
[2026-03-04 19:42:11 +0000] [12] [INFO] Booting worker with pid: 12
	
dirtviz-api
March 4, 2026, 11:42
	
[2026-03-04 19:42:11 +0000] [1] [ERROR] Worker (pid:10) was sent SIGKILL! Perhaps out of memory?
	
dirtviz-api
March 4, 2026, 11:42
	
[2026-03-04 19:42:10 +0000] [1] [CRITICAL] WORKER TIMEOUT (pid:10)
```

--- a/backend/api/resources/cell_data.py
+++ b/backend/api/resources/cell_data.py
@@ -2,13 +2,8 @@ from flask import request, jsonify
 from flask_restful import Resource
 import pandas as pd
 from ..schemas.get_cell_data_schema import GetCellDataSchema
-from ..models.power_data import PowerData
-from ..models.teros_data import TEROSData
-from ..models.sensor import Sensor
-from functools import reduce
 from io import StringIO
 from celery import shared_task
-import numpy as np
 
 get_cell_data = GetCellDataSchema()
 
@@ -17,52 +12,19 @@ get_cell_data = GetCellDataSchema()
 def stream_csv(self, request_args):
     """Creates a task that queries cell data and writes into buffer
 
+    jmadden173: Temperarily converted this to a blank download.
+
     Arguments:
     request_args -- arguments of the request being made
     """
-    v_args = get_cell_data.load(request_args)
-    cell_ids = v_args["cellIds"].split(",")
-    for cell_id in cell_ids:
-        teros_data = pd.DataFrame(
-            TEROSData.get_teros_data_obj(
-                cell_id,
-                resample=v_args["resample"],
-                start_time=v_args["startTime"],
-                end_time=v_args["endTime"],
-            )
-        )
-        power_data = pd.DataFrame(
-            PowerData.get_power_data_obj(
-                cell_id,
-                resample=v_args["resample"],
-                start_time=v_args["startTime"],
-                end_time=v_args["endTime"],
-            )
-        )
-        sensor_data = pd.DataFrame(
-            Sensor.get_sensor_data_obj(
-                name="phytos31",
-                cell_id=cell_id,
-                measurement="voltage",
-                resample=v_args["resample"],
-                start_time=v_args["startTime"],
-                end_time=v_args["endTime"],
-            )
-        )
-        data_frames = [teros_data, power_data, sensor_data]
 
-        df_merged = reduce(
-            lambda left, right: pd.merge(left, right, on=["timestamp"], how="outer"),
-            data_frames,
-        ).fillna("void")
+    csv_buffer = StringIO()
 
-        csv_buffer = StringIO()
+    df = pd.DataFrame()
 
-        # buffer writes to memory
-        for chunk in np.array_split(df_merged, 10):
-            chunk.to_csv(csv_buffer, index=False, header=(csv_buffer.tell() == 0))
+    df.to_csv(csv_buffer, index=False)
 
-        return csv_buffer.getvalue()
+    return csv_buffer.getvalue()
 
 
 class Cell_Data(Resource):

--- a/frontend/src/pages/dashboard/Dashboard.jsx
+++ b/frontend/src/pages/dashboard/Dashboard.jsx
@@ -567,6 +567,26 @@ function Dashboard() {
                 <Typography variant='h6' color='text.secondary'>
                   Please select one or more cells above to view environmental sensor data
                 </Typography>
+                <Box
+                  sx={{
+                    backgroundColor: "#d32f2f",
+                    color: "white",
+                    px: 2,
+                    py: 1.5,
+                    textAlign: "center",
+                    fontFamily: "sans-serif"
+                  }}
+                >
+                  CSV export is currently non-functional. See the issue for updates:{" "}
+                  <a
+                    href="https://github.com/jlab-sensing/ENTS-backend/issues/668"
+                    target="_blank"
+                    rel="noreferrer"
+                    style={{ color: "#ffffff", textDecoration: "underline", fontWeight: "bold" }}
+                  >
+                    GitHub Issue #668
+                  </a>
+              </Box>
               </Box>
             </Box>
           ) : showNoDataMessage ? (

